package.json

```diff
@@ -849,6 +849,7 @@
     "p-limit": "^3.0.1",
     "p-map": "^4.0.0",
     "p-retry": "^4.2.0",
+    "p-settle": "4.1.1",
```

Comment:
I intentionally pulled in version 4.1.1 here because in 5.x they use ESM and I was getting these errors:

`Error [ERR_REQUIRE_ESM]: require() of ES Module`

https://github.com/sindresorhus/p-settle/commit/27a9d0801eb3a4b33edf5b145ff5b5e229255545